prov/verbs/src/ep_rdm/verbs_av_ep_rdm.c

```diff
@@ -590,6 +594,7 @@ fi_ibv_rdm_av_tbl_idx_to_conn_add_new_conn(struct fi_ibv_rdm_ep *ep,
 	struct fi_ibv_rdm_av_entry *av_entry =
 		fi_ibv_rdm_av_tbl_idx_to_av_entry(ep, addr);
 	if (av_entry) {
+		pthread_mutex_lock(&av_entry->conn_lock);
```

Comment:
mutex must be released in error path below

Reply:
Done